### ISSUE TITLE
Default versionType

### ISSUE THREAD

**Issue by raboof** [iamamoose/Vulnogram#44]
We currently have a default versionType of `custom`. Usually it would be more precise to use `maven` or `semver`. Would there be a way to default maven projects to `maven` and other projects to `semver`?

**Comment by iamamoose** [comment]
how do we know maven projects?  would we prefer it to be blank?

**Comment by raboof** [comment]
> how do we know maven projects? would we prefer it to be blank?

Leaving it blank is a bit tedious, since it doesn't change... perhaps we should default to 'semver' and have a list of projects where we default to 'maven' instead, similar to `pmcswithsecurityemails`?

**Comment by iamamoose** [comment]
I was hoping we'd get rid of pmcswithsecurityemails eventually though and rely on infra-provided data.... we should find a way for something else to give us the info we need rather than any hardcoded list.

**Comment by iamamoose** [comment]
(however i don't mind defaulting to semver in the meantime)

**Comment by raboof** [comment]
> we should find a way for something else to give us the info we need rather than any hardcoded list.

yeah that would be better... I guess getting the DOAP and assuming programming-language=java implies Maven would be a fairly close heuristic, but also not trivial ;)
